docs/configuration-examples.md

```diff
@@ -0,0 +1,266 @@
+    "keyAlias": "keyalias",
+    "keystoreP12Path": "/path/to/auth-keystore.p12",
+    "keystorePassword": "keystorepassword",
+
+    // domains to which this config should be applied to.
+    // defaults to ["mastercard.com","api.ethocaweb.com"]
+    "appliesTo": [
+      "mastercard.com",
+      "api.ethocaweb.com"
+    ]
+  }
+}
+```
+
+## Mastercard Encryption
+```json
+{
+  "mastercard": {
+    "consumerKey": "000000000000000000000000000000000000000000000000!000000000000000000000000000000000000000000000000",
+    "keyAlias": "keyalias",
+    "keystoreP12Path": "/path/to/auth-keystore.p12",
+    "keystorePassword": "keystorepassword",
+
+    // domains to which this config should be applied to.
+    // defaults to ["mastercard.com","api.ethocaweb.com"]
+    "appliesTo": [
+      "mastercard.com",
+      "api.ethocaweb.com"
+    ],
+
+    "encryptionConfig": {
+      "paths": [
+        {
+          "path": "/*",
+          "toEncrypt": [
+            {
+              // path to the element to be encrypted in request JSON.
+              // use "$" for encrypting the whole request.
+              "element": "path.to.element.to.be.encrypted",
+              // path to object where encryption fields are to be stored in request JSON.
+              // use "$" for the root of the JSON object.
+              "obj": "path.to.encrypted.output.element"
+            }
+          ],
+          "toDecrypt": [
+            {
+              // path to object with encryption fields in response JSON.
+              // use "$" for the root of the JSON.
+              "element": "path.to.element.to.be.decrypted", 
+
+              // path to element where decrypted fields are to be stored in the response JSON.
+              // use "$" for the root of the JSON object.
+              "obj": "path.to.decryption.output"
+            }
+          ]
+        }
+      ],
+      "oaepPaddingDigestAlgorithm": "SHA-256",
+      "dataEncoding": "hex",  // "hex" or "base64"
+      "ivFieldName": "iv",
+      "encryptedKeyFieldName": "encryptedKey",
+      "encryptedValueFieldName": "encryptedValue",
+      "oaepHashingAlgorithmFieldName": "oaepHashingAlgorithm",
+      "publicKeyFingerprintFieldName": "publicKeyFingerprint",
+      "publicKeyFingerprintType": "certificate", // "certificate" or "publicKey"
+      "publicKeyFingerprint": "0000000000000000000000000000000000000000000000000000000000000000",
+      "encryptionCertificate": "/path/to/encryption-certificate.pem",
+      "keyStore": "/path/to/decryption-keystore.p12",
+      "keyStoreAlias": "decryption-keyalias",
+      "keyStorePassword": "decryption-keystorepassword"
+    }
+  }
+}
+```
+### Mastercard Encryption Real world example  
+
+This is a real world configuration for the [Donations API](https://developer.mastercard.com/donations/documentation/service_config/), which uses Mastercard Encryption.
```

Comment:
Please remove the mention of Donations API, Once petstore is ready, we can use it. For now, just state This is a configuration example, for an API, uses Mastercard Encryption

Reply:
removed.